ms/MSOper/MSMetaData.cc

```diff
@@ -40,6 +41,12 @@
 #include <casacore/tables/TaQL/TableParse.h>
 #include <casacore/tables/Tables/TableProxy.h>
 
+#include <utility>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
```

Comment:
this is not needed when you are not using the openmp C-api (function calls starting with ´omp_`)